### PR TITLE
Adding groovy script to run the corefxlab build on PR and daily

### PR DESCRIPTION
**Pull request by sokket**
Adding groovy script to run the corefxlab build on PR and daily

/cc @mmitche

### CODE REVIEW

**Comment by sokket** [comment]
test ci please

**Comment by sokket** [comment]
@mmitche I don't seem to have access to the Settings page for this repo...any chance you do?

**Comment by sokket** [comment]
test ci please

File: netci.groovy [hunk 1]
```diff
@@ -0,0 +1,34 @@
+// Import the utility functionality.
+
+import jobs.generation.Utilities;
+import jobs.generation.InternalUtilities;
```
Comment (mmitche) [inline netci.groovy]: Remove this line.  You only need the Utilities functionality since this is a public repro

File: netci.groovy [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+def project = 'dotnet/corefxlab'
+
+// Define build strings
+def debugBuildString = '''build.cmd debug'''
+
+// Generate the builds for debug 
+
+def windowsDebugJob = job(InternalUtilities.getFullJobName(project, 'windows_debug', false)) {
```
Comment (mmitche) [inline netci.groovy]: InternalUtilities -> Utilities

**Comment by mmitche** [comment]
You can simplify this a lot with the example here: https://github.com/dotnet/dotnet-ci/blob/master/docs/simple-netci.groovy

**Comment by mmitche** [comment]
Added the hooks

**Comment by sokket** [comment]
test ci please

**Comment by sokket** [comment]
@mmitche Changed the groovy script to be the changed sample one instead

**Comment by mmitche** [comment]
LGTM

**Comment by sokket** [comment]
Thanks!

**Pull request merged**